### PR TITLE
Fix grammatical mistakes and one other issue

### PR DESCRIPTION
**Pull request by YounesMDA**
Removed unnecessary commas, changed the phrasing of some sentences to be more understandable and added one sentence to be more indicative of what is being explained.

### CODE REVIEW

File: docs/csharp/language-reference/keywords/base.md [hunk 1]
```diff
@@ -12,25 +12,25 @@ ms.assetid: 8b645dbe-1a33-49b8-8716-1c401f9a5ea5
...
 
-The `base` keyword is used to access members of the base class from within a derived class:
+The `base` keyword is used to access members of the base class from within a derived class. Use it if you want to:
 
 - Call a method on the base class that has been overridden by another method.
 
 - Specify which base-class constructor should be called when creating instances of the derived class.
 
-A base class access is permitted only in a constructor, an instance method, or an instance property accessor.
+The base class access is permitted only in a constructor, in an instance method and in an instance property accessor.
```
Comment (BillWagner) [inline docs/csharp/language-reference/keywords/base.md:21]: Our style guide does call for the Oxford comma:

```suggestion
The base class access is permitted only in a constructor, in an instance method, and in an instance property accessor.
```

File: docs/csharp/language-reference/keywords/base.md [hunk 2]
```diff
@@ -12,25 +12,25 @@ ms.assetid: 8b645dbe-1a33-49b8-8716-1c401f9a5ea5
...
 
 ## Example 1
 
-In this example, both the base class, `Person`, and the derived class, `Employee`, have a method named `Getinfo`. By using the `base` keyword, it is possible to call the `Getinfo` method on the base class, from within the derived class.
+In this example, both the base class `Person` and the derived class `Employee` have a method named `Getinfo`. By using the `base` keyword, it is possible to call the `Getinfo` method of the base class from within the derived class.
 
 [!code-csharp[csrefKeywordsAccess#1](~/samples/snippets/csharp/VS_Snippets_VBCSharp/csrefKeywordsAccess/CS/csrefKeywordsAccess.cs#1)]
 
-For additional examples, see [new](new-modifier.md), [virtual](virtual.md), and [override](override.md).
+For additional examples, see [new](new-modifier.md), [virtual](virtual.md) and [override](override.md).
```
Comment (BillWagner) [inline docs/csharp/language-reference/keywords/base.md:33]: also an Oxford comma:

```suggestion
For additional examples, see [new](new-modifier.md), [virtual](virtual.md), and [override](override.md).
```

**Review by BillWagner — COMMENTED** [review]
Thanks @YounesMDA 

Overall, these changes look great. Our style guide mandates the Oxford comma, so I made two suggestions. Once you accept those, I'll merge this.

**Review by BillWagner — APPROVED** [review]
Thanks @YounesMDA 

I'll :shipit: now.

**Comment by YounesMDA** [comment]
Thanks!

**Pull request merged**